recipes/ucc/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+./autogen.sh
+./configure --prefix=$PREFIX --with-ucx=$PREFIX --with-cuda=$PREFIX
+make -j $CPU_COUNT
+make install
```

Comment:
Is there a make check?

Reply:
There is but it doesn't seem to be doing anything useful.